requirements/edx/base.in

```diff
@@ -125,6 +125,7 @@ python-openid
 python-saml==2.4.0
 pyuca==1.1                          # For more accurate sorting of translated country names in django-countries
 reportlab==3.1.44                   # Used for shopping cart's pdf invoice/receipt generation
+rest-condition>=1.0.3,<2.0          # DRF's recommendation for supporting complex permissions
```

Comment:
Is it recommended to include the <2.0, or should we just wait until something breaks and keep on trucking if nothing ever breaks?